source/security.txt

```diff
@@ -0,0 +1,21 @@
+.. _mongoid-security:
+
+================
+Secure Your Data
+================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: ruby framework, odm, security
+
+.. toctree::
+   :caption: Secure Your Data
+   
+   Client-Side Field Level Encryption </security/encryption>
```

Comment:
S: consider shortening this to just In-Use Encryption to adhere to the TOC relabeling guidelines